lib/plugins/aws/package/compile/events/eventBridge/index.js

```diff
@@ -58,8 +60,11 @@ class AwsCompileEventBridgeEvents {
     const { provider } = service;
     const { compiledCloudFormationTemplate } = provider;
     const iamRoleStatements = [];
+    const { eventBridge: options } = provider;
+    const useCloudFormation = options ? options.useCloudFormation : false;
```

Comment:
Is this `useNativeCloudFormation` or `useCloudFormation`? I think your document is talking about `useNativeCloudFormation`.